jdi-light-bdd/src/main/java/com/epam/jdi/bdd/JDISteps.java

```diff
@@ -126,6 +122,17 @@ public void setAttribute(String name, String attrName, String attrValue) {
 		BaseUIElement el = getUI(name);
 		el.setAttribute(attrName, attrValue);
 	}
+
+	@When("I go forward")
```

Comment:
- add start/end line anchor (^/$ respectively)
- add (?:I |) at the beginning of action steps

Reply:
Resolved